deploy/controller/csi-addons-config.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: csi-addons-config
+  # replace the namespace with the namespace where the operator is deployed.
+  namespace: csi-addons-system
+data:
+  "reclaim-space-timeout": "5m"
+  "max-concurrent-reconciles": "50"
```

Comment:
match the values for what we have in default , if the user just creates this configmap should not have any impact unless he/she modifies the values